db/migrate/20220519055836_add_timeline_algorithm_to_user_invitation.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimelineAlgorithmToUserInvitation < ActiveRecord::Migration[6.0]
+  def change
+    add_column :course_user_invitations, :timeline_algorithm, :integer
```

Comment:
Note: For future development when timeline algorithm setting is included in a course user invitation

Reply:
@qreoct it's good to also include more information in your PR description on related backend setup that you have created for future development (eg in this case - Allow user invitation to specify the PT algorithm at the individual user level during the invite phase, both in file upload and individual invite)